tests/datadriven/__init__.py

```diff
@@ -194,6 +200,24 @@ def assertRaisesWith(self, exceptionType):
                 "exception of type {} not raised".format(
                     exceptionType.__name__))
 
+    def assertAlmostEqual(self, a, b, ndigits=7):
+        """
+        Assert that a and b are equal within ndigits digits
+        """
+        if round(a-b, ndigits) != 0:
```

Comment:
Hi Maciek,

I think you might be looking to have it like this:

```Python
def round_to_sigdigs( a_number, sig=7):
	return round(a_number, sig-int(floor(log10( a_number )))-1)
	
def assertAlmostEqual(self, a, b, ndigits=7):
	"""
	Assert that a and b are equal within ndigits digits
	"""
	if ( round_to_sigdigs(a, ndigits) - round_to_sigdigs(b, ndigits) ) != 0:
		 message = "{} and {} not equal within {} digits".format(
			 a, b, ndigits)
	raise AssertionError(message)
```

This is referenced from this:

http://stackoverflow.com/questions/3410976/how-to-round-a-number-to-significant-figures-in-python
	
Example:

```Python
>>> round_to_sigdigs(a,7) - round_to_sigdigs(b,7)
0.0
```
	
`p